client/packages/invoices/src/OutboundShipment/DetailView/OutboundLineEdit/OutboundLineEdit.tsx

```diff
@@ -68,6 +68,7 @@ export const OutboundLineEdit: React.FC<ItemDetailsModalProps> = ({
   const { Modal } = useDialog({ isOpen, onClose, disableBackdrop: true });
   const [currentItem, setCurrentItem] = useBufferState(item);
   const [isAutoAllocated, setIsAutoAllocated] = useState(false);
+  const [okDisabled, setOkDisabled] = useState(false);
```

Comment:
disabling ok until the debounce has happened. without it, you could click ok before the allocation has happened and get 0 stock allocated